src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="post"
+      data-qa="big">
```

Comment:
closing brackets should also be on a newline, fix it everywhere
```suggestion
    <form
      action="/"
      method="post"
      data-qa="big"
    >
```